RoT/Protocol/RFCs/0001-Transport_Agnosticism.md

```diff
@@ -0,0 +1,149 @@
+* Name: Transport_Agnosticism
```

Comment:
Didn't the template have a field for RFC number, or was that supposed to added upon completion of the PR?

Reply:
I think we should just file a separate PR to add it to the template and then add it to all currently-checked-in RFCs? WDYT? Feel free to file an issue and assign it to me.

Reply:
That sounds like a good approach.